components/sidebar/sidebarlayerview.tsx

```diff
@@ -0,0 +1,59 @@
+import Expandable from 'components/expandable';
+import React from 'react';
+import { IceCharts } from './icecharts';
+import { SARImagery } from './sarimagery';
+import { SeaIce } from './seaiceconcentration';
+import { SideBarAction, SideBarActionKind } from './sidebar.types';
+
+interface SideBarLayerViewProps {
+  onChange: (params: SideBarAction) => void;
+}
+
+enum EXPANDABLES {
+  SARIMAGERY = 'SARIMAGERY',
+  SEAICE = 'SEAICE',
+  ICECHARTS = 'ICECHARTS',
+}
+
+const SidebarLayerView: React.FC<SideBarLayerViewProps> = ({ onChange }) => {
+  const [expanded, setExpanded] = React.useState<EXPANDABLES>(EXPANDABLES.SARIMAGERY);
+
+  return (
+    <div className="space-y-1">
+      <Expandable
```

Comment:
The Layer group item (e.g sar imagery) should have a number after the layer group name representing how many layers are selected on the group 

<img width="171" alt="image" src="https://user-images.githubusercontent.com/9701591/167621147-8121fdd2-43af-4500-a633-d984b1e4bf3d.png">